### ISSUE TITLE
Can't login via oauth on ownCloud 7 server

### ISSUE THREAD

**Issue by tallship** [owncloud/android#1585]
### Expected behaviour

When trying to logon to ownCloud 7 server that is oauth enabled, i am unable to do so worth android client app.
### Actual behaviour

It just keeps failing and failing and of i want tho login I am forced tho use a web browser I my phone instead of the ownCloud android app
### Steps to reproduce
1. Try loghing in with oath via Android app
2. Continuous errors
3. Give up and just login via web browser
### Environment data

Android version:
Lollipop

Device model: lg stylo

Stock or customized system:

Rooted
ownCloud app version: 1.90

Also doesn't work with ownCloud beta 20160120 either.

ownCloud server version: 7.02 i think

**Comment by debyst** [comment]
I had a very similar behaviour: my android client app didn't accepted my passwort, although i entered it correctly. however, i was able to log in to owncloud via webbrowser over my phone.

android-version: 5.1
owncload android app version: 1.9.1
device: fairphone2 (not rooted)

By chance I figured out what the solution to my problem was: i used characters like ö,ä,ü in my password, which apperently was not correctly encoded in the app. 
I was able to reproduce this behaviour by changing the password several times, using ä,ö,ü once and without using them the other time.
